addons/product/models/product_template.py

```diff
@@ -5,7 +5,7 @@
 import logging
 from collections import defaultdict
 
-from odoo import api, fields, models, tools, _, SUPERUSER_ID
+from odoo import api, fields, models, tools, _, SUPERUSER_ID, Command
```

Comment:
```suggestion
from odoo import Command, SUPERUSER_ID, _, api, fields, models, tools
```
Since we are touching that line...
`ord('A') < ord('_') < ord('a')`